build/pkgs/ipython/distros/conda.txt

```diff
@@ -1 +1,2 @@
-ipython
+ipython>=7.13.0,<8.9.0
+# ipython >= 8.9.0 requires prompt_toolkit too new for Sage
```

Comment:
This change is obsolete with #36659

Reply:
do we have a plan to switch to an up to date prompt_toolkit? I vaguely recall a discussion on this, but I can't find it.

Reply:
#35251

Reply:
It's still required, see https://github.com/sagemath/sage/actions/runs/6804955526/job/18503541665?pr=36678.

Reply:
That run didn't apply any CI fixes because https://github.com/sagemath/sage/actions/runs/6804955526/job/18503541665?pr=36678#step:3:7